app/assets/javascripts/discourse/app/models/permission-type.js

```diff
@@ -2,23 +2,27 @@ import I18n from "I18n";
 import discourseComputed from "discourse-common/utils/decorators";
 import EmberObject from "@ember/object";
 
+export function buildPermissionDescription(id) {
+  let key = "";
```

Comment:
This is quite a few more lines than creating a `const PERMISSION_DESCRIPTIONS = { 1: 'full', 2: 'create_post, 3: ... }` then doing `return I18n.t("permission_types." + PERMISSION_DESCRIPTIONS[key]);`

Reply:
sounds good. I just moved the function to export it, but it is a good opportunity to refactor :+1: